store_test.go

```diff
@@ -31,15 +30,6 @@ func testNewController(t *testing.T, provider, url string) (NetworkController, e
 	return New(cfgOptions...)
 }
 
-func TestBoltdbBackend(t *testing.T) {
```

Comment:
@mavenugo ... libnetwork on Solaris does not support 'host' networking used by this test.